apps/platform/src/app/app.component.html

```diff
@@ -1,108 +1,75 @@
+         <ng-container id="items">
+            <biosimulations-dropdown-menu-item heading="Browse Simulation Runs" [routerLink]="['/runs']">
+            </biosimulations-dropdown-menu-item>
+            
+            <biosimulations-dropdown-menu-item heading="Run A Simulation" [routerLink]="['/runs', 'new']">
+            </biosimulations-dropdown-menu-item>
+         </ng-container>
+      </biosimulations-hover-open-menu>
 
-        <biosimulations-dropdown-menu-item heading="Run A Simulation" [routerLink]="['/runs', 'new']">
-        </biosimulations-dropdown-menu-item>
-      </ng-container>
-    </biosimulations-hover-open-menu>
+      <biosimulations-hero-banner-util-button></biosimulations-hero-banner-util-button>
 
-    <biosimulations-hover-open-menu>
-      <biosimulations-topbar-menu-item heading="Utilities" icon="construction" id="trigger">
-      </biosimulations-topbar-menu-item>
-      <ng-container id="items">
-        <biosimulations-dropdown-menu-item heading="Convert a file" [routerLink]="['/utils', 'convert-file']">
-        </biosimulations-dropdown-menu-item>
+      <biosimulations-hover-open-menu [href]="config.docsUrl" [disabled]="true" target="docs">
+         <biosimulations-topbar-menu-item heading="Learn" icon="idea" id="trigger"> 
+         </biosimulations-topbar-menu-item>
+      </biosimulations-hover-open-menu>
 
-        <biosimulations-dropdown-menu-item heading="Create a project" [routerLink]="['/utils', 'create-project']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate a model" [routerLink]="['/utils', 'validate-model']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item
-          heading="Validate a simulation"
-          [routerLink]="['/utils', 'validate-simulation']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate metadata" [routerLink]="['/utils', 'validate-metadata']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate a project" [routerLink]="['/utils', 'validate-project']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item
-          heading="Validate a simulator"
-          href="https://github.com/biosimulators/Biosimulators/issues/new?assignees=&labels=Validate%2Fsubmit+simulator&template=ValidateOrSubmitASimulator.yml&title=%5BSimulation+capabilities%5D%3A+"
-          target="githubissues">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Suggest a simulator" [routerLink]="['/utils', 'suggest-simulator']">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="BioSimulations REST API" [href]="config.apiUrl">
-        </biosimulations-dropdown-menu-item>
-      </ng-container>
-    </biosimulations-hover-open-menu>
-
```

Comment:
you removed the utilities navigation from the platform app?  is this intentional?